### ISSUE TITLE
Disable copying current path for new documents

### ISSUE THREAD

**Issue by Alhadis** [atom/status-bar#101]
A picture's worth a thousand words, so I figure this one says all:

<img width="124" alt="screen shot 2015-08-27 at 11 29 16 pm" src="https://cloud.githubusercontent.com/assets/2346707/9521896/c8e568d2-4d14-11e5-98d2-b84b23c6a671.png">

I can't see anybody wanting to copy "untitled" to their clipboard when creating a new document; heck, they might even click on it by accident and wipe their system clipboard (which I nearly did, being as tired as I am...)

Maybe limit this behaviour to saved documents only?

**Comment by Alhadis** [comment]
**UPDATE:** Okay, the scope of this issue extends to non-documents, too:

![1](https://cloud.githubusercontent.com/assets/2346707/9592000/efa54142-5082-11e5-9832-32e0aa7fe837.png)

**Comment by batjko** [comment]
Thanks for the ping, @50Wliu.

@Alhadis Can I ask what your expectation was when you clicked on the "untitled" text? Why did you click it and what did you expect to happen?

**Comment by Alhadis** [comment]
I clicked it by mistake, as I explained in the OP. And technically, nothing should happen.

**Comment by batjko** [comment]
Ok so that's one way to go, i.e. that nothing should happen. 

Another one, and the reason why it behaves like this right now, is that it is consistent that a click on that text copies whatever is displayed. 

Consistency of expectation being the main reasoning here, i.e. that people normally wouldn't click that text on purpose, unless they expect to copy it.

However, I also see the point that copying "untitled" or "Settings" doesn't make a lot of sense. 
Perhaps changing the tool-tip to something generic in that case, like "Not a valid path", so that even if nothing gets copied, people get some informative feedback...

**Comment by Alhadis** [comment]
I'd say just remove the visual cue (underline) and the pointer cursor so there's no incentive for the user to click it and expect a result.

**Comment by mnquintana** [comment]
> I'd say just remove the visual cue (underline) and the pointer cursor so there's no incentive for the user to click it and expect a result.

I agree with this. I think this is a case where maintaining consistency actually leads to a confusing user experience, because there is essentially no use case for copying a "path" from the status bar that isn't a real path.

I would've said before that the solution is to disable path copying for everything but open file buffers, but with https://github.com/atom/status-bar/pull/100 now I think we'd have to disable it only for "non-path" items (ie. any string that doesn't match a path or URL pattern).

**Comment by batjko** [comment]
Yes, that makes sense to me. I'll see if I can take some time to add that in quickly.

> so there's no incentive for the user to click it and expect a result

"Don't you touch that... no... leave it alone... you know it doesn't make sense, just... now you did! Why? Why did you just click this thing that says nothing but "untitled"... What in the world did you expect to happen here? Do I really have to remove the incentive for you to click on it. Fuck it, you clicked on your last "Settings" link, Mister!"

**Comment by Alhadis** [comment]
LOL, something like that. ;p
